src/index/CompressedBlockPrefiltering.h

```diff
@@ -87,6 +108,9 @@ class RelationalExpression : public PrefilterExpression {
       : referenceId_(referenceId) {}
 
   std::unique_ptr<PrefilterExpression> logicalComplement() const override;
+  bool operator==(const PrefilterExpression& other) const override;
+  std::unique_ptr<PrefilterExpression> clone() const override;
+  std::string info([[maybe_unused]] size_t depth) const override;
```

Comment:
```suggestion
  std::string info(size_t depth) const override;
```
If necessary, `[[maybe_unused]]` should only be in the implementation/definition, not in the declaration.